packages/react-charting/src/components/Legends/Legends.base.tsx

```diff
@@ -35,10 +35,9 @@ interface ILegendItem extends React.ButtonHTMLAttributes<HTMLButtonElement> {
 }
 
 export interface ILegendState {
-  selectedLegend: string;
   activeLegend: string;
   isHoverCardVisible: boolean;
-  /** Set of legends selected when multiple selection is allowed */
+  /** Set of legends selected, both for multiple selection and single selection */
   selectedLegends: { [key: string]: boolean };
```

Comment:
This boolean value seems to be redundant with only true value

Reply:
Yes, I also wonder why not `string[]`?

Reply:
This seems to be existing code. We can let it be for now.